Makefile

```diff
@@ -31,7 +31,7 @@ NVARCH ?= -arch=sm_70 \
 	  -gencode=arch=compute_75,code=sm_75 \
 	  -gencode=arch=compute_75,code=compute_75
 
-CFLAGS    ?= -fPIC -O3 -funroll-loops -march=native
+CFLAGS    ?= -fPIC -O3 -funroll-loops -march=native -fopenmp
```

Comment:
Yes, I added it because the timing result indicating that `onedim_fseries_kernel` being the performance bottleneck. I removed the flag in a670fdc and will create another pull request for moving the part of the codes onto GPU so that we don't introduce the dependency of OpenMP.

Reply:
Yes, I added it because the timing result indicating that `onedim_fseries_kernel` being the performance bottleneck. I remove it in a670fdc and will create another pull request for moving the part of the codes to GPU so that we don't introduce the dependency of OpenMP.

Reply:
Ah cool. Sounds good.